dpbench/benchmarks/rodinia/gaussian/gaussian_sycl_native_ext/gaussian_sycl/_gaussian_kernel.hpp

```diff
@@ -0,0 +1,58 @@
+// SPDX-FileCopyrightText: 2022 - 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+#include <CL/sycl.hpp>
+
+using namespace sycl;
+
+template <typename FpTy>
+void gaussian_kernel_1(FpTy *m_device,
+                       const FpTy *a_device,
+                       int Size,
+                       int t,
+                       sycl::nd_item<3> item_ct1)
+{
+    if (item_ct1.get_local_id(2) +
+            item_ct1.get_group(2) * item_ct1.get_local_range().get(2) >=
+        Size - 1 - t)
+        return;
+    *(m_device +
+      Size * (item_ct1.get_local_range().get(2) * item_ct1.get_group(2) +
+              item_ct1.get_local_id(2) + t + 1) +
+      t) = *(a_device +
+             Size * (item_ct1.get_local_range().get(2) * item_ct1.get_group(2) +
+                     item_ct1.get_local_id(2) + t + 1) +
+             t) /
+           *(a_device + Size * t + t);
```

Comment:
Minor nitpicking. It would be visually easier to comprehend `a_device[Size*t + t]` as supposed to `*(a_device + Size * t + t)`. Further it would be consistent with `gaussian_kernel_2` and also the dpex_k impl.

Reply:
modified as per the comment.